src/mongodb/scores.service.ts

```
@@ -2603,4 +2638,83 @@ export class ScoresService {
+
+  public async getComprehensionScore(subSessionId: string, language: string) {
+    const sessions = await this.getSubSessionScores(subSessionId, language);
+    const comprehensionScores: any[] = [];
+    sessions.forEach((session: any) => {
+      if (session.comprehension !== undefined) {
+        comprehensionScores.push(session.comprehension);
+      }
+    });
+    let overallScore = 0;
+    let isComprehension = false;
+    if(comprehensionScores.length>0){
```

ESLint

[error] Replace `(comprehensionScores.length>0)` with `·(comprehensionScores.length·>·0)·`

(prettier/prettier)